gitbootcamp.rst

```diff
@@ -28,26 +28,26 @@ relevant to CPython's workflow.
 Forking CPython GitHub Repository
 ---------------------------------
 
-You'll only need to do this once.
+You will only need to do this once.
 
 1. Go to https://github.com/python/cpython.
 
 2. Press ``Fork`` on the top right.
 
 3. When asked where to fork the repository, choose to fork it to your username.
 
-4. Your fork will be created at https://github.com/<username>/cpython.
+4. Your forked CPython repository will be created at https://github.com/<username>/cpython.
```

Comment:
What about :
```suggestion
4. Your CPython repository forked will be created at https://github.com/<username>/cpython.
```

Reply:
This is not proper grammar:
> Your CPython repository forked ...

"Your forked CPython repository" is more correct, but an alternative could be:

> Your fork of the CPython repository

Reply:
Sorry for the delay of me response. 

> This is not proper grammar:

Sorry for the noise I am not English speaker, I believe that was a good change. Thanks and sorry.

Reply:
> Sorry for the noise I am not English speaker, I believe that was a good change. Thanks and sorry.

No problem! It's always better to try to better to provide feedback than not submit any at all. It's appreciated either way. (: